crates/red_knot_python_semantic/resources/mdtest/conditionals.md

```diff
@@ -0,0 +1,15 @@
+# Conditionals
```

Comment:
Can you please move this test to `crates/red_knot_python_semantic/resources/mdtest/narrow/conditionals_not_equals.md` and adapt the structure to the what the other files in this folder have?